x-pack/plugin/core/src/test/java/org/elasticsearch/license/XPackLicenseStateTests.java

```diff
@@ -201,246 +200,44 @@ public void testMonitoringAckNotBasicToBasic() {
-        assertAllowed(BASIC, true, s -> s.checkFeature(Feature.LOGSTASH), false);
-    }
-
-    public void testLogstashInactive() {
-        assertAllowed(BASIC, false, s -> s.checkFeature(Feature.LOGSTASH), false);
-        assertAllowed(TRIAL, false, s -> s.checkFeature(Feature.LOGSTASH), false);
-        assertAllowed(GOLD, false, s -> s.checkFeature(Feature.LOGSTASH), false);
-        assertAllowed(PLATINUM, false, s -> s.checkFeature(Feature.LOGSTASH), false);
-        assertAllowed(STANDARD, false, s -> s.checkFeature(Feature.LOGSTASH), false);
-    }
-
-    public void testJdbcDefaults() {
-        XPackLicenseState licenseState = TestUtils.newTestLicenseState();
-        assertThat(licenseState.checkFeature(XPackLicenseState.Feature.JDBC), is(true));
-    }
-
-    public void testJdbcBasic() {
-        XPackLicenseState licenseState = TestUtils.newTestLicenseState();
-        licenseState.update(BASIC, true, null);
-        assertThat(licenseState.checkFeature(XPackLicenseState.Feature.JDBC), is(false));
-    }
-
-    public void testJdbcStandard() {
-        XPackLicenseState licenseState = TestUtils.newTestLicenseState();
-        licenseState.update(STANDARD, true, null);
-
-        assertThat(licenseState.checkFeature(XPackLicenseState.Feature.JDBC), is(false));
-    }
-
-    public void testJdbcStandardExpired() {
-        XPackLicenseState licenseState = TestUtils.newTestLicenseState();
-        licenseState.update(STANDARD, false, null);
-
-        assertThat(licenseState.checkFeature(XPackLicenseState.Feature.JDBC), is(false));
-    }
-
-    public void testJdbcGold() {
-        XPackLicenseState licenseState = TestUtils.newTestLicenseState();
-        licenseState.update(GOLD, true, null);
-
-        assertThat(licenseState.checkFeature(XPackLicenseState.Feature.JDBC), is(false));
-    }
-
-    public void testJdbcGoldExpired() {
-        XPackLicenseState licenseState = TestUtils.newTestLicenseState();
-        licenseState.update(GOLD, false, null);
-
-        assertThat(licenseState.checkFeature(XPackLicenseState.Feature.JDBC), is(false));
-    }
-
-    public void testJdbcPlatinum() {
-        XPackLicenseState licenseState = TestUtils.newTestLicenseState();
-        licenseState.update(PLATINUM, true, null);
-
-        assertThat(licenseState.checkFeature(XPackLicenseState.Feature.JDBC), is(true));
-    }
-
-    public void testJdbcPlatinumExpired() {
-        XPackLicenseState licenseState = TestUtils.newTestLicenseState();
-        licenseState.update(PLATINUM, false, null);
-
-        assertThat(licenseState.checkFeature(XPackLicenseState.Feature.JDBC), is(false));
-    }
-
-    public void testSqlAckAnyToTrialOrPlatinum() {
-        assertAckMessages(XPackField.SQL, randomMode(), randomTrialOrPlatinumMode(), 0);
-    }
-
-    public void testSqlAckTrialOrPlatinumToNotTrialOrPlatinum() {
-        assertAckMessages(XPackField.SQL, randomTrialOrPlatinumMode(), randomBasicStandardOrGold(), 1);
-    }
```

Comment:
These test the acknowledgement messages between licenses for specific features. I think they are not covered by the new tests and should therefore be retained.